cli/image-scanning-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,68 @@
+[id="image-scanning-by-using-the-roxctl-cli"]
+:_mod-docs-content-type: ASSEMBLY
+= Image scanning by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: image-scanning-by-using-the-roxctl-cli
+
+toc::[]
+
+[role="_abstract"]
+You can scan images stored in image registries, including cluster local registries such as the {ocp} integrated image registry by using the `roxctl` CLI.
+
+[id="scanning-images-in-a-remote-cluster_{context}"]
+== Scanning images using a remote cluster
+
+By specifying the appropriate cluster in the delegated scanning configuration or the cluster parameter described in the following procedure, you can scan images from cluster local registries by using a remote cluster.
+
+[IMPORTANT]
+====
+For more information about how to configure delegated image scanning, see xref:../operating/examine-images-for-vulnerabilities.adoc#configuring-delegated-image-scanning_examine-images-for-vulnerabilities[Configuring delegated image scanning].
+====
+
+.Procedure
+
+* Run the following command to scan the specified image in a remote cluster:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl image scan --image=_<image_registry>_/_<image_name>_ --cluster=_<cluster_detail>_ [flags]<1>
+----
++
```

Comment:
Either [splitting the command over multiple lines or a definition list](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks) is the appropriate way to add callouts here

Reply:
@dcaravel I will be splitting command as per the doc guidelines for better readability.
$ roxctl image scan \
  --image=<image_registry>/<image_name> \// <1>
  --cluster=<cluster_detail> \// <2>
  [flags] <3>